sim/Makefile

```diff
@@ -53,7 +53,8 @@ override SCALA_BUILDTOOL_DEPS += $(SBT_THIN_CLIENT_TIMESTAMP)
 SBT_CLIENT_FLAG = --client
 endif
 
-SBT ?= java $(JAVA_OPTS) -jar $(rocketchip_dir)/sbt-launch.jar $(SBT_OPTS) $(SBT_CLIENT_FLAG)
+SBT_BIN ?= sbt
+SBT = $(SBT_BIN) $(SBT_CLIENT_FLAG)
```

Comment:
We are using the `sbt` binary (this is really a script) that is provided through conda because it fixes an issue where SBT thin client functionality (i.e. the `SBT_CLIENT_FLAG`) was broken (errors would return error code `0`). This also aligns Chipyard and FireSim to use the same set of arguments to set SBT/java opts (`SBT_OPTS`, `JAVA_TOOL_OPTIONS`).